### PR TITLE
Map some block entity constants

### PR DESCRIPTION
**Pull request by YanisBft**
Tried my best to name them correctly

### CODE REVIEW

File: mappings/net/minecraft/block/PointedDripstoneBlock.mapping [hunk 1]
```diff
@@ -8,6 +8,8 @@ CLASS net/minecraft/class_5689 net/minecraft/block/PointedDripstoneBlock
 	FIELD field_28056 FRUSTUM_SHAPE Lnet/minecraft/class_265;
 	FIELD field_28057 MIDDLE_SHAPE Lnet/minecraft/class_265;
 	FIELD field_28058 BASE_SHAPE Lnet/minecraft/class_265;
+	FIELD field_31211 MAX_WATER_DRIP_CHANCE F
```
Comment (enbrain) [inline mappings/net/minecraft/block/PointedDripstoneBlock.mapping]:
```suggestion
	FIELD field_31211 WATER_DRIP_CHANCE F
```
This is a chance itself. The `dripTick` method checks if a random number is less than `0.17578125`. The `dripChance` parameter is actually a sampled value.

File: mappings/net/minecraft/block/PointedDripstoneBlock.mapping [hunk 2]
```diff
@@ -8,6 +8,8 @@ CLASS net/minecraft/class_5689 net/minecraft/block/PointedDripstoneBlock
...
+	FIELD field_31212 MAX_LAVA_DRIP_CHANCE F
```
Comment (enbrain) [inline mappings/net/minecraft/block/PointedDripstoneBlock.mapping]:
```suggestion
	FIELD field_31212 LAVA_DRIP_CHANCE F
```

File: mappings/net/minecraft/block/entity/BannerBlockEntity.mapping [hunk 3]
```diff
@@ -3,6 +3,7 @@ CLASS net/minecraft/class_2573 net/minecraft/block/entity/BannerBlockEntity
 	FIELD field_11772 customName Lnet/minecraft/class_2561;
 	FIELD field_11773 patternListNbt Lnet/minecraft/class_2499;
 	FIELD field_11774 baseColor Lnet/minecraft/class_1767;
+	FIELD field_31296 MAX_PATTERNS_COUNT I
```
Comment (enbrain) [inline mappings/net/minecraft/block/entity/BannerBlockEntity.mapping]:
```suggestion
	FIELD field_31296 MAX_PATTERN_COUNT I
```
This looks like the maximum number of elements in the `Patterns` list.

File: mappings/net/minecraft/block/entity/SculkSpreadManager.mapping [hunk 4]
```diff
@@ -1,4 +1,6 @@
 CLASS net/minecraft/class_7128 net/minecraft/block/entity/SculkSpreadManager
+	FIELD field_37610 MAX_CHARGES I
```
Comment (enbrain) [inline mappings/net/minecraft/block/entity/SculkSpreadManager.mapping]:
```suggestion
	FIELD field_37610 MAX_CHARGE I
```

File: mappings/net/minecraft/block/entity/PistonBlockEntity.mapping [hunk 5]
```diff
@@ -57,7 +57,9 @@ CLASS net/minecraft/class_2669 net/minecraft/block/entity/PistonBlockEntity
 		ARG 0 box
 		ARG 1 entity
 	METHOD method_23672 moveEntity (Lnet/minecraft/class_2350;Lnet/minecraft/class_1297;DLnet/minecraft/class_2350;)V
+		ARG 0 direction
 		ARG 1 entity
+		ARG 4 oppositeDirection
```
Comment (enbrain) [inline mappings/net/minecraft/block/entity/PistonBlockEntity.mapping]: I don't think this is accurate. Two usages pass the same direction.

File: mappings/net/minecraft/block/PointedDripstoneBlock.mapping (hunk 1 above)
Comment (YanisBft) [reply mappings/net/minecraft/block/PointedDripstoneBlock.mapping]: Makes sense, changed

File: mappings/net/minecraft/block/entity/PistonBlockEntity.mapping (hunk 5 above)
Comment (YanisBft) [reply mappings/net/minecraft/block/entity/PistonBlockEntity.mapping]: now better named

**Review by apple502j — APPROVED** [review]

**Review by liach — APPROVED** [review]
can unpick later

**Review by enbrain — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**